moveit_ros/planning/planning_scene_monitor/test/current_state_monitor_test.cpp

```diff
@@ -145,11 +145,22 @@ TEST_F(CurrentStateMonitorTest, StateUpdateTest)
 
 TEST_F(CurrentStateMonitorTest, IncrementalTimeStamps)
 {
-  sendJointStateAndWait(js_a);
-  EXPECT_EQ(js_a.header.stamp, csm->getCurrentStateTime());
+  sensor_msgs::JointState js_b_new;
+  js_b_new.name = { "b-c-joint" };
+  js_b_new.position = { 0.25 };
+  js_b_new.velocity = { 0.25 };
+  js_b_new.header.stamp = ros::Time{ 10.5 };
 
   sendJointStateAndWait(js_b);
-  EXPECT_EQ(js_a.header.stamp, csm->getCurrentStateTime()) << "older stamp made csm jump backwards in time";
```

Comment:
Instead of modifying existing tests, you should only add new ones to test the new functionality.

Reply:
@rhaschke could you respond to the above concern on the change of behavior? I guess we should mention about that in MIGRATION/CHANGELOG if we would make the change.